diff --git a/node_modules/@synonymdev/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java b/node_modules/@synonymdev/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java
index 5c82167..c5e0907 100644
--- a/node_modules/@synonymdev/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java
+++ b/node_modules/@synonymdev/react-native-keychain/android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageBase.java
@@ -409,6 +409,7 @@ abstract public class CipherStorageBase implements CipherStorage {
       if (null == isStrongboxAvailable || isStrongboxAvailable.get()) {
         if (null == isStrongboxAvailable) isStrongboxAvailable = new AtomicBoolean(false);
 
+        /*
         try {
           secretKey = tryGenerateStrongBoxSecurityKey(alias);
 
@@ -416,6 +417,7 @@ abstract public class CipherStorageBase implements CipherStorage {
         } catch (GeneralSecurityException | ProviderException ex) {
           Log.w(LOG_TAG, "StrongBox security storage is not available.", ex);
         }
+        */
       }
     }
 
